.github/workflows/main.yml

```diff
@@ -74,6 +74,17 @@ jobs:
         run: >
           echo "CHANGED_FILES=$(echo $(git diff --name-only ${{ github.event.pull_request.base.sha }}..${{ github.event.pull_request.head.sha }} -- | grep \.py))" >> $GITHUB_ENV
 
+      - if: env.CHANGED_FILES
```

Comment:
I found the commit subject a bit misleading, it looked like some fixes in the Python code to fix import errors reported by pylint.  Maybe something like "install requirements" would be clearer.

Reply:
Are you still experimenting with this?

Reply:
Updated the commit message.